ssh/DOCS.md

```diff
@@ -6,50 +6,57 @@ Follow these steps to get the add-on installed on your system:
 
 1. This addon is only visible to "Advanced Mode" users. To enable advanced mode, go to **Profile** -> and turn on **Advanced Mode**.
 2. Navigate in your Home Assistant frontend to **Supervisor** -> **Add-on Store**.
-3. Find the "SSH server" add-on and click it.
+3. Find the "Terminal & SSH" add-on and click it.
 4. Click on the "INSTALL" button.
 
-## How to use
-
-You need enable the port for external access. You can just enter 22 as value or any other
-value like you want. This recommend to add login credentials.
-
-To use this add-on, you must have a private/public key to log in.
-To generate them, follow the [instructions for Windows][keygen-windows]
-and [these for other platforms][keygen]. It is possible to set a password for
-login since version 2.0 but for high security use private/public keys.
 
-You can not run both variants at the same time. Enabling login via keys, will
-disable password login.
+## How to use
 
-1. Add a ssh key to  `authorized_keys` or set a `password` in the add-on configuration.
-2. Start the add-on.
-3. Connect to your device using your preferred SSH client and use `root` as
-   the username.
+Once installed (following the steps above) you need to
+* enable external access by selecting which TCP port to open up (see Network section), and
+* supply login credentials,
+* start the add-on.
 
-After logging in, you will find yourself in this add-on’s container.
-The Home Assistant configuration directory is mounted on the path `/config`.
+You can then connect to the port specified, and using the username `root`
+you will end up in this add-on's container. The Home Assistant configuration 
+directory is mounted on the path `/config`.
 
-Try it out using:
+This add-on comes bundled with [The Home Assistant CLI](https://www.home-assistant.io/hassio/commandline/). Try it out using:
 
 ```bash
 ha help
 ```
 
+### General Note on Security
+
+You are strongly recommended to use private/public keys to log in. 
```

Comment:
Why?

Reply:
The intent with my contribution was to clean up this page, as I discovered the documentation was not up to date with the implementation when I - as a normal user - installed this core Add-On.

In doing so (updating documentation to match behavior present in latest stable release), I have no reason to question the original authors' suggestions - perhaps they've implemented logging of passwords in clear text? While I happen to be a software developer, I have not read the implementation, and I hope this project accepts documentation improvements without having to have read, understood and need to reason about the applicability of the documented recommended use of the software it documents.

On the other hand, as a professional developer, I would argue that keys stored on an external card (or similar device), with two factor protection (both key and the external device are individually protected) provides a much stronger scheme, compared to username+password combination that is always sent over the network. Sending your public key over the network, compared to a password, is always preferable. (It is unclear what process HA suggests be put in place for rotating the server's keys, and help humans understand what "the server's key fingerprint has changed" really means, but I believe that's out of scope for this add-on, and this page as well.)

Some parts of the internet tend to agree, such as discussed here: https://security.stackexchange.com/questions/69407/why-is-using-an-ssh-key-more-secure-than-using-passwords

Now, I can personally agree that making a recommendation may be "over the top". I'll reword this section along the lines of "if you enable external access to you machine, using keys is preferred over password based authentication".

Reply:
Thanks, I was very well aware why. However, the user that reads that maybe not. Hence why question.

Reply:
Ah. From your "Why?" comment, I didn't understand you wanted me to educate or motivate the recommendation to the end user. I assume the new wording is sufficient (please reopen if you feel otherwise).